governance/permissionless-proposals.md

```diff
@@ -0,0 +1,90 @@
+| Council Member Elections      | Elections to replace or renew signers in a given cohort.                                                                                                                                                 | Any approved address by OP Foundation | Ranked Choice            | - Input valid proposal type<br>- Proposer is an approved address<br>- No funds are transferred<br>- No state changes | - Gathered needed delegate approvals<br>- Voting cycle enforcement |
+| Governance Fund (Missions)    | Used to support Collective development and ecosystem growth via the Mission framework.                                                                                                                   | Permissionless                        | Approval                 | - Input valid proposal type<br>- Requested amount does not exceed the OP Cap                                         | - Gathered needed delegate approvals<br>- Voting cycle enforcement |
+| Council Budget                | Allocation of funds for governance operations.                                                                                                                                                           | Permissionless                        | Approval                 | - Input valid proposal type<br>- Requested amount does not exceed the OP Cap                                         | - Gathered needed delegate approvals<br>- Voting cycle enforcement |
+
+# The Workflow
+
+The general workflow would be:
+
+1. **Draft Proposal:** A delegate starts by creating a Draft proposal and posting it on the optimism governance forum. At this stage, the proposal is open to the entire community for feedback. 
+2. **Proposal Finalization**: After the proposal has been drafted and shared on the forum, delegates provide feedback, ask clarifying questions, and propose changes. Once the delegate incorporates this input and the proposal is refined, the delegate updates its status to "Final Draft". At this point, the delegate must create an on-chain transaction to create the proposal metadata record in the `ProposalValidator` by calling `submitProposal()`, which can be done either through Etherscan or Agora UI. This call emits a `ProposalSubmitted` event, which includes the `proposalId` and other relevant data. Agora listens for this event and uses it to fetch and display the finalized proposal on its UI, allowing delegates to begin the sign-off process.
+    
+```mermaid
+flowchart LR
+    Delegate([Delegate]) -->|Submit DRAFT proposal| OPGov1[OPGov Forum]
+    Community([Community]) -->|Provide feedback| OPGov1
+    OPGov1 -->|Submit FINAL proposal| OPGov2[OPGov Forum]
```

Comment:
```suggestion
    Delegate([Delegate]) -->|Submit DRAFT proposal| OPGov[OPGov Forum]
    Community([Community]) -->|Provide feedback| OPGov
    OPGov -->|`submitProposal()` once FINAL| OnchainGov[ProposalValidator]
```

It should be this right?  I see the same diagram as current in markdown but I think just a typo, as the figma comment confirms "final == onchain"